jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -147,19 +147,23 @@ def test_multiprocessing():
         model1.data[2, j + 1, :, :] = model1.data[2, j, :, :] + delta_plane1 + delta_plane2
     model1.data = np.round(model1.data + np.random.normal(0, 5, (nints, ngroups, ncols, nrows)))
 
+    model2 = model1.copy()
+    gain2 = gain.copy()
+    rnoise2 = rnoise.copy()
```

Comment:
Just curious: what's the purpose or need for copies of these?

Reply:
Processing modifies input data, so if these copies are not done, the single processing and multiprocessing will process slightly different data.

Reply:
OK, so at least we know that it won't cause any problems/crashes in the Jenkins environment. It just won't give a useful result.